tests/unit/components/settings/settings.spec.js

```diff
@@ -58,7 +58,7 @@ describe('NavBar', () => {
   test('Connected Edge device version is shown', () => {
     const localEdgeVersion = require('@/../package.json').version
```

Comment:
@vickywane package.json holds the PWA version, not the edge device version.

Reply:
I had previous thinking that the About and Settings page showed the version of the connected edge device. 

I will correct it this.